extra/Dockerfile

```
@@ -50,7 +36,7 @@ RUN echo "deb http://snapshot.debian.org/archive/debian/$(date +%Y%m%d) bullseye
     pip install /tmp/motioneye && \
     # Cleanup
     rm -rf motioneye && \
-    apt-get purge --yes python3-setuptools python3-wheel && \
+    apt-get purge --yes python3-setuptools python3-wheel python3-pip && \
     apt-get autoremove --yes && \
```

```suggestion
    apt-get --yes autopurge && \
```
This also removes the config files of removed packages.